src/main/java/me/moodcat/api/SongAPI.java

```diff
@@ -157,19 +158,22 @@ private VAVector adjustSongVector(final ClassificationRequest classification, fi
      * @author JeremybellEU
      */
     @AllArgsConstructor
+    @NoArgsConstructor
     protected static class ClassificationRequest {
 
         /**
          * The valence for the song.
          */
         @Getter
-        private final double valence;
+        @Setter
```

Comment:
If you have both `@Setter` and `@Getter` on all attributes of the class, why not use `@Data` on the class?